### ISSUE TITLE
Wrong choice of words for translation

### ISSUE THREAD

**Issue by sergeykorobeynikov** [retypeapp/retype#586]
Not quite a correct translation for
```yml
locale: ru
```
The preposition "By" before Author in this case is better not to translate at all, or "Автор", but for "Создано" it is somehow not for a native speaker.

**Comment by geoffreymcgill** [comment]
The Retype translation files are hosted within the following open-source GitHub project:

https://retypeapp.github.io/retype-translations/
https://github.com/retypeapp/retype-translations/tree/main

The language specific source files are at https://github.com/retypeapp/retype-translations/tree/main/src

The specific `By` comment is available at:

https://github.com/retypeapp/retype-translations/blob/a9b27defafd7c523946ce5817c66f84fd8b25129/src/UserInterface.ru.resx#L345

```xml
  <data name="PageMeta_By_Label" xml:space="preserve">
    <value>Создано</value>
    <comment>By</comment>
  </data>
```

Revisions to the source translation files are always welcome.

I will get confirmation on the change to `Автор` or just an empty string and we will include the update in the Retype v3.1 release.

**Comment by geoffreymcgill** [comment]
I created a new issue in the [retype-translations](https://github.com/retypeapp/retype-translations) project to track this change, see https://github.com/retypeapp/retype-translations/issues/4.

**Comment by geoffreymcgill** [comment]
We went with an empty string `<value></value>`. The change will be included in the upcoming Retype v3.1 release.

**Comment by sergeykorobeynikov** [comment]
I think this will be true for all languages. The only time when it is necessary to indicate the status of the author of the text is when the text is a translation from another language.

Regards!
